control_plane/src/compute.rs

```
@@ -407,7 +407,7 @@ impl PostgresNode {
 
     // Always destroy pgdatadir when we stop postgres
     pub fn stop(&self) -> Result<()> {
-        self.pg_ctl(&["stop"])?;
+        let _ = self.pg_ctl(&["stop"]);
```

Is there a reason to ignore error here?